handlers/validator.go

```diff
@@ -1541,21 +1547,116 @@ func ValidatorHistory(w http.ResponseWriter, r *http.Request) {
 	}
 
 	currentEpoch := services.LatestEpoch() - 1
-	// for an exited validator we show the history until his exit
+	var extraEpochs uint64 = 0
```

Comment:
**Minor:** For the sake of readability I suggest to not use a generic prefix lile `extra` for the added handling.
Maybe something like `postExit`?
`extraEpochs` => `postExitEpochs`
`extraItemLength` => `postExitItemLength`

If you agress, please also consider changing the comment here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/b03ce154335997cba7cc60f9ff24042536e719ea/handlers/validator.go#L1575

Reply:
refactored